manifests/init.pp

```diff
@@ -66,6 +66,15 @@
   String[1] $facts_file_owner = 'root',
   String[1] $facts_file_group = 'root',
   Optional[Stdlib::Filemode] $facts_file_mode = '0644',
+  Stdlib::Absolutepath $facter_conf_dir = '/etc/puppetlabs/facter',
```

Comment:
Please document the new params above with `@param` statements

Reply:
Added docs.